lib/settings-panel.coffee

```diff
@@ -120,7 +120,7 @@ class SettingsPanel extends CollapsibleSectionPanel
 
   set: (name, value) ->
     if @options.scopeName
-      if value is undefined
+      if value is undefined or `value == atom.config.get(name)`
```

Comment:
@thomasjo I literally added a comment just after you commented, which explains why.